### ISSUE TITLE
Can't install 3DN BIP!!! Blender crash when try to install! Linux OS

### ISSUE THREAD

**Issue by FidoAnh** [tingjoybits/Brush_Manager#8]
I can use Install Pillow in Window but when I try this button in Linux, my blender crash!
I attached blender console crash and blender crash file for more information! Thank you!
![Screenshot_20220513_065652](https://user-images.githubusercontent.com/103020170/168186167-aa304036-3e34-4a2f-bea4-f86b0d8b2c54.png)
[blender.crash.txt](https://github.com/tingjoybits/Brush_Manager/files/8683170/blender.crash.txt)

**Comment by tingjoybits** [comment]
Indeed, thanks for the report! investigating the problem... D: I will try to fix it as soon as possible!

**Comment by tingjoybits** [comment]
Type this command in the console to uninstall the pillow package for now, so you could use the Blender again: 
_(Your path to blender folder)_/blender-3.2.0/3.2/python/bin/python3.10 -m pip uninstall -y Pillow

**Comment by tingjoybits** [comment]
Unfortunately, I have to turn off the support of the 3DN Bip library on the Linux systems, until the issue with Pillow package in Blender will be resolved. Fortunately, the speed of icon previews has been improved in the recent Blender versions, so personally, I would not stress much about the current situation. The update will be out soon.
